src/App.jsx

```diff
@@ -8,7 +8,9 @@ import { getTodos } from './api';
 class App extends React.Component {
   state = {
     todos: [],
-    selectedUserId: 0,
+    selectedUserId: '',
+    query: '',
+    status: '',
```

Comment:
you could create state in TodoList component, and store this values there. All app shouldn't know about query and status